src/twig/package.json

```diff
@@ -1,12 +1,18 @@
 {
   "name": "design-system-twig",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha __tests__/*.js __tests__/**/*.js"
   },
   "keywords": [],
   "author": "",
-  "license": "Apache"
+  "license": "Apache",
```

Comment:
Is it `Apache` or `Apache 2.0`? I feel like we've been swapping back and forth.

Reply:
It throws me an error when I use "Apache" and so I think it should be "Apache 2.0"